go/test/endtoend/utils/mysql.go

```diff
@@ -114,7 +114,7 @@ func createInitSQLFile(mysqlDir, ksName string) (string, error) {
 		return "", err
 	}
 	defer f.Close()
-
+	_, _ = f.WriteString("SET GLOBAL super_read_only='OFF';")
```

Comment:
In these test we create Mysql without any vttablets (i-e barecluster) and then try to create database so I have to set super_read_only to false given its not going through flow of typical vttablet.

Reply:
let's catch the error and fail if it is not nil.

Reply:
done.